### DISCUSSION TITLE
Suggested New Interaction for Knife and Chopping Block: Slice and Julienne.

### DISCUSSION THREAD

**Discussion by tariqk** [vectorwing/FarmersDelight#839] [Ideas]
Just putting this up, since while I know while other add-on mods can add this in, but I feel like this could really benefit the core mod itself.

I propose that Farmer's Delight add knife interactions to vegetables, starting with vanilla vegetables:

* Potatoes ⇒ Sliced Potatoes ⇒ Julienne Potatoes
* Carrots ⇒ Sliced Carrots ⇒ Julienne Carrots
* Beets ⇒ Sliced Beets ⇒ Julienne Beets

Furthermore, you should be able to smelt the byproducts of these items into the following items:

* Sliced Potatoes ⇒ Sautéed Potatoes
* Sliced Carrots ⇒ Sautéed Carrots
* Sliced Beets ⇒ Sautéed Beets
* Julienne Potatoes ⇒ French Fries
* Julienne Carrots ⇒ _Carrots Julienne_
* Julienne Beets ⇒ Borscht? (probably as a pot recipe)

Alternatively, some other plants, both vanilla and Farmer's Delight veg, can be interacted on the chopping board to, you know…

* Onions ⇒ Chopped Onions
* Mushrooms ⇒ Sliced Mushrooms ⇒ Chopped Mushrooms
* Tomatoes ⇒ Sliced Tomatoes ⇒ Chopped Tomatoes

Which can be incorporated into dips, relishes and sauces where necessary.

Okay, part of the reason I'm bringing this up is that I'm surprised that Farmer's Delight doesn't have a way of making french fries, especially considering how the interaction of putting a potato on a chopping board is _just there_.

**Comment by Aceplante** [comment]
Then cut potatoes into fries, which can then be cooked? You dont need all that other stuff
